Hey there! Thank you for taking the time to do this. 🙂 

You've definitely caught some sloppy refactoring on my part there. It's acted as a prompt for me to set up a Python linter -- I'm working on a new machine and it shows. 😅 So thank you for that. I might take the time to integrate a linter into the project.

The **import** changes look **great** and I'm happy to accept those. 👍 

I'm not sure I want to accept the **test** changes though -- here's why:
1. While the use of unittest-style assertions may not be recommended for PyTest projects, **this is _genuinely_ a unittest project**, so it makes perfect sense to be using those assertions. 
    
    Put another way: PyTest is an external dependency and isn't pulled in by the project, so right now `import pytest` fails. 😅

    Maybe in future I might look at switching to PyTest if it can bring maintainability/legibility/functionality improvements, but for now I'd like to stick to unittest. (Please don't make a PR switching the project to PyTest unless you're comfortable it might not be accepted and you're doing it for fun and/or the learning experience!)

2. Using the base `assert` seems like a step back in usability when a test fails. This is because `assert` only receives a `True` or `False` value and can't print any info about why a mismatch occurred.

    Here's a comparison -- first the `assert`:

    ```
        assert result['crdhldBillAmt'] == 7.54288
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    AssertionError
    ```

    Then `assertEqual`:

    ```
        self.assertEqual(result['crdhldBillAmt'], 7.54288)
        ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    AssertionError: 7.54287 != 7.54288
    ```

    Notice how the unittest-style assertion gives some extra context, telling us what the mismatch is -- something missing in the base `assert` that would force us to debug or add logging.

    I do agree on one thing: the `self.assertEqual(result, True)` is redundant. For consistency with other tests, I'd probably swap it out for another unittest assertion, `self.assertTrue()` -- perhaps not as useful, but it is consistent:

    ```
        self.assertTrue(result)
        ~~~~~~~~~~~~~~~^^^^^^^^
    AssertionError: False is not true
    ```

Anyway, I hope what I've written is useful. Likewise, I appreciate you taking time out of your day to submit this PR. 😄 

Do the changes I've requested make sense? Are you comfortable making the changes I've requested, or would you like me to do them? 🛠️

--- a/mc/tests/repository/test_mastercard.py
+++ b/mc/tests/repository/test_mastercard.py
@@ -24,7 +24,7 @@ class TestMastercardSettle(unittest.TestCase):
             transaction_currency='USD',
         )
 
-        assert result['crdhldBillAmt'] == 7.54287
+        self.assertEqual(result['crdhldBillAmt'], 7.542870)
 
     def test_throws_on_bad_status(self):
         httpretty.register_uri(
@@ -70,7 +70,7 @@ class TestMastercardRatesAvailable(unittest.TestCase):
 
         result = mastercard.rates_available('2018-06-03')
 
-        assert result
+        self.assertTrue(result)
 
     def test_success_false(self):
         httpretty.register_uri(
@@ -81,7 +81,7 @@ class TestMastercardRatesAvailable(unittest.TestCase):
 
         result = mastercard.rates_available('2018-06-03')
 
-        assert not result
+        self.assertFalse(result)
 
     def test_throws_on_bad_status(self):
         httpretty.register_uri(

--- a/mc/tests/repository/test_mastercard.py
+++ b/mc/tests/repository/test_mastercard.py
@@ -5,7 +5,6 @@ import httpretty
 from requests import HTTPError
 
 from repository import mastercard
-import pytest
 
 
 @httpretty.activate
@@ -34,7 +33,7 @@ class TestMastercardSettle(unittest.TestCase):
             status=400
         )
 
-        with pytest.raises(HTTPError):
+        with self.assertRaises(HTTPError):
             mastercard.settle(
                 bank_fee_percentage=0,
                 card_currency='GBP',
@@ -91,7 +90,7 @@ class TestMastercardRatesAvailable(unittest.TestCase):
             status=400
         )
 
-        with pytest.raises(HTTPError):
+        with self.assertRaises(HTTPError):
             mastercard.rates_available('2018-06-03')
 
     @staticmethod

